styles.css

```diff
@@ -0,0 +1,85 @@
+#background {
+  background-size: cover;
+  background-image: url("background-img.png");
+  width: 100%;
+  height: 100vh;
+}
+
+header {
+  display: flex;
+  width: 100%;
+  align-content: space-between;
+  justify-content: space-between;
+}
```

Comment:
- [ ] The content of your header is not properly aligned vertically at the center because it is missing the property `align-item: center`. KIndly replace align-content with it 👍🏽 

![image](https://user-images.githubusercontent.com/63473584/212425813-80383b1c-5124-4627-a670-8479a1c19b63.png)